packages/application-shell/src/configure-apollo.ts

```diff
@@ -100,7 +69,8 @@ export const createApolloClient = () =>
   new ApolloClient({
     link,
     cache: new InMemoryCache({
-      fragmentMatcher,
+      // https://www.apollographql.com/docs/react/data/fragments/#defining-possibletypes-manually
+      // possibleTypes: [],
```

Comment:
This is the only change that I don't know if "it just works" or not, and we probably need to "test" to ensure things work.

As [documented here](https://www.apollographql.com/docs/react/migrating/apollo-client-3-migration/#cache-improvements), the new Apollo client cache comes with some breaking changes.

Reply:
As I understood it using the `IntrospectionFragmentMatcher` is similar to using `possibleTypes: []`. Should we maybe allow an application to pass the `possibleTypes` and merge them in with a static list of pre-defined static types (if ever defined) here?